features/step_definitions/web_steps.rb

```diff
@@ -102,6 +102,10 @@ def with_scope(locator)
   attach_file(field, File.expand_path(path))
 end
 
+Then ("shoud see clickable text {string}") do |string|
```

Comment:
Sorry to be picky, but this step can only be used to check for linkedin links which will not be obvious to future developers.  Can you either have the href be a value that's passed in or change the step text to make it obvious it's for linkedin only?  Also, you left out the "I" at the beginning so your tests are still failing.  Are you running these tests locally before pushing?

Reply:
I just pushed to github to save my changes I didnt mean to have it be reviewed again yet, I am still not done. My computer has been acting funny so I just wanted to push everything so nothing gets lost. I am running the test locally and it fails them locally, I just pushed to keep save my progress so far not because the test passed.

Reply:
I will change the step text so its more obvious that it is just for linkedin